lib/ffmpeg/qsv/encoder.py

```diff
@@ -32,6 +32,10 @@ def gen_quality_opts(self):
       return " -global_quality {quality}"
     return " -preset {quality}"
 
+  def gen_hevc_main10sp_opts(self):
+    if self.profile in ["main10sp"]:
+      return " -main10sp 1"
+
```

Comment:
Just add `-main10sp 1` in mapprofile return value.

Reply:
updated, thanks